### DISCUSSION TITLE
您好 我是一位纯小白 在搭建网站的时候遇到了一些问题

### DISCUSSION THREAD

**Discussion by spacemeowfish** [YunYouJun/hexo-theme-starry#333] [Q&A 疑难杂症]
![image](https://user-images.githubusercontent.com/110675284/183252934-7d8e127c-3cc8-44b7-9545-50a5e6b0a883.png)
您好 在按照您的搭建网站步骤一步步进行的时候 遇到了这个问题 并不能继续下去
在网上查询试着在git文件夹的config文件里加了username和useremail也没有作用

**Comment by YunYouJun** [comment]
<img width="201" alt="image" src="https://user-images.githubusercontent.com/25154432/183305178-9bc1ee7e-3868-4b4a-816d-20e8fbc2948a.png">

用户名和仓库名看起来不一致。

**Reply by spacemeowfish** [reply]
![image](https://user-images.githubusercontent.com/110675284/183435288-786a1751-bfaf-4375-9c34-1c5c04e11d22.png)
您好··在今天又尝试了一次后还是不太行。。。

**Reply by YunYouJun** [reply]
<img width="620" alt="image" src="https://user-images.githubusercontent.com/25154432/185793040-00f3a254-d821-46d7-af11-fadce4b76a5c.png">

email 地址错误。
